locale/en/security.md

```diff
@@ -21,13 +20,12 @@ These updates will be sent at least every five days; in practice, this is more l
 
 The Node.js project engages in an official bug bounty program for security researchers and responsible public disclosures.
 
-The program is managed through the HackerOne platform at [https://hackerone.com/nodejs](https://hackerone.com/nodejs) with further details.
+The program is managed through the HackerOne platform at <https://hackerone.com/nodejs> with further details.
```

Comment:
This text was pre-existing, but it didn't make any grammatical sense as-is. I think line 22 (blank) should be removed so this sentence becomes part of the preceeding paragraph, and should be rewritten as:

> The program is managed through the HackerOne platform, see <https://hackerone.com/nodejs> for further details.

Reply:
I split that into two sentences to be grammatically-correct, but good catch.